internal/commands/sbomtest/sbomloader.go

```diff
@@ -0,0 +1,112 @@
+package sbomtest
+
+import (
+	"bufio"
+	"errors"
+	"fmt"
+	"io"
+	"os"
+	"strings"
+)
+
+type foundMatch struct {
+	first  bool
+	second bool
+}
+
+type sbomMatchTuple struct {
+	first  string
+	second string
+}
+
+func (t sbomMatchTuple) key() string {
+	return t.first + "-" + t.second
+}
+
+var supportedSBOMFormatTuples []sbomMatchTuple = []sbomMatchTuple{
+	{"CycloneDX", "bomFormat"},
+	{"SPDXRef-DOCUMENT", `"spdxVersion"`},
+}
+
+func IsSupportedSBOMFormat(inputFile io.Reader) (bool, error) {
```

Comment:
i would suggest we do not do this. the sbom-test service already contains logic for this, this is a repitition on the client side which i don't believe we need. at the most, i would suggest we change this to something like `IsJSON` that just tests if the file contents can be `json.Unmarshall`ed.